### PR TITLE
New About Page Design

### PR DESCRIPTION
**Pull request by Damola18**
# Improvement of the About Page UI 

This PR 
- revamps the outlook of the about page
- updates all the sections on the about page including the history and mission
- updates employee profile
- update scripts to allow only pnpm

Hero Section

image

Mission Section
Screenshot 2024-06-27 at 04 11 40

Team Section

Screenshot 2024-06-27 at 09 04 12

History Section
Screenshot 2024-06-27 at 09 01 24

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2783

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- This animation is not working. Also it is quite small. The lines should also be pink (brand color)
image

- This section shouldn't have a drop shadow.
image

- On the history section, there should be more spacing between the header and the content.
image

- the pattern that's meant to be below the number is missing
image

- There is a scroll bar indicator that shouldn't be there
image

- I realized the headings for each history is too big. so i will be adjusting this in the design too.
image

- Let's add more padding between the content and footer too.
image

**Comment by Damola18** [comment]
> * This animation is not working. Also it is quite small. The lines should also be pink (brand color)
> 
> image
> * This section shouldn't have a drop shadow.
> 
> image
> * On the history section, there should be more spacing between the header and the content.
> 
> image
> * the pattern that's meant to be below the number is missing
> 
> image
> * There is a scroll bar indicator that shouldn't be there
> 
> image
> * I realized the headings for each history is too big. so i will be adjusting this in the design too.
> 
> image
> * Let's add more padding between the content and footer too.
> 
> image

Oh, let me check, something must have broken here, the images shows here. And yeah, i noticed the scrollbar and the pattern as well, i just added it. Let me check the images first please

**Comment by Softdev1** [comment]
I think there is a conflict between the design and implementation in the UI for some session 

- This is not what is in the design, kindly fix it 
image

- The dark mode text for this box should be white, not black 
image


Please I think it would be nice, let's try and resolve these issues

**Comment by Tolu-Mals** [comment]
Nice work so far.

Here's what I noticed:

- The orbiting circle should be bigger than what we have at the moment, it should be like this
Screenshot 2024-06-27 at 10 22 29 AM

- Also like Steve pointed out, the two big solid circles in the middle should not be the same size, on the design one is slightly smaller.

- The background color for the solid circles are different from the design. They should be more transparent such that the inner dashed lines should be clearly visible through the circles.

- The size of the white outline circles in the bigger solid pink circles are much smaller in the design, and one of them is actually a rounded square and not a circle

**Comment by Damola18** [comment]
> Nice work so far.
> 
> Here's what I noticed:
> 
> * The orbiting circle and IQ logo should be bigger than what we have at the moment, it should be like this
>   Screenshot 2024-06-27 at 10 22 29 AM
> * Also like Steve pointed out, the two big solid circles in the middle should not be the same size. On the design, one of the circles is slightly smaller.
> * The background color for the solid circles are different from the design. They should be more transparent such that the inner dashed lines should be clearly visible through the circles.
> * The size of the white outline circles in the bigger solid pink circles are much smaller in the design, and one of them is actually a rounded square and not a circle



> Nice work so far.
> 
> Here's what I noticed:
> 
> * The orbiting circle and IQ logo should be bigger than what we have at the moment, it should be like this
>   Screenshot 2024-06-27 at 10 22 29 AM
> * Also like Steve pointed out, the two big solid circles in the middle should not be the same size. On the design, one of the circles is slightly smaller.
> * The background color for the solid circles are different from the design. They should be more transparent such that the inner dashed lines should be clearly visible through the circles.
> * The size of the white outline circles in the bigger solid pink circles are much smaller in the design, and one of them is actually a rounded square and not a circle



> I think there is a conflict between the design and implementation in the UI for some session
> 
> * This is not what is in the design, kindly fix it
> 
> image
> * The dark mode text for this box should be white, not black
> 
> image
> Please I think it would be nice, let's try and resolve these issues



> I think there is a conflict between the design and implementation in the UI for some session
> 
> * This is not what is in the design, kindly fix it
> 
> image
> * The dark mode text for this box should be white, not black
> 
> image
> Please I think it would be nice, let's try and resolve these issues

Thanks for the observations, will check out and fix that

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
It's a lot better than what we had before. Great job.
In addition to those pointed out:

- the space between the cards is huge and the border effect (magic cards) is missing.
it also makes it look weird in md breakpoint.
image
image

- The pink card is still larger in width than the rest of the page.
image
- This should remain the same even in dark mode.
image

**Comment by Simplypheyie96** [comment]
> Thanks very much, working on the changes highlighted.
> @Simplypheyie96 for the circles, can we have the different circle for the other one then. I only have one for both (for the ones you masked)

I have helped create those circles, but you will need to adjust their sizes in code to fit the orbiting circle.

**Comment by Damola18** [comment]
> > Thanks very much, working on the changes highlighted.
> > @Simplypheyie96 for the circles, can we have the different circle for the other one then. I only have one for both (for the ones you masked)
> 
> I have helped create those circles, but you will need to adjust their sizes in code to fit the orbiting circle.

Thanks

**Comment by Damola18** [comment]
https://github.com/EveripediaNetwork/ep-ui/assets/50779080/c7afebfd-8338-471d-bd22-d041f657d72b

@Simplypheyie96 , this is for the card

**Comment by Simplypheyie96** [comment]
> https://github.com/EveripediaNetwork/ep-ui/assets/50779080/c7afebfd-8338-471d-bd22-d041f657d72b
> 
> @Simplypheyie96 , this is for the card border effect .

Does it trigger only on hover?
It should work like the one we have on iq gpt

**Comment by Damola18** [comment]
> > Screen.Recording.2024-06-27.at.11.08.17.mov
> >     
> >   
> > 
> >   
> > 
> >   
> > 
> > 
> > @Simplypheyie96 , this is for the card border effect .
> 
> Does it trigger only on hover? It should work like the one we have on iq gpt

You mean The magic card effect or border beam effect?

**Comment by Simplypheyie96** [comment]
> You mean The magic card effect or border beam effect?

this is what we are using. Border beam.
https://magicui.design/docs/components/border-beam

**Comment by Damola18** [comment]
> I think there is a conflict between the design and implementation in the UI for some session
> 
> * This is not what is in the design, kindly fix it
> 
> image
> * The dark mode text for this box should be white, not black
> 
> image
> Please I think it would be nice, let's try and resolve these issues



> > You mean The magic card effect or border beam effect?
> 
> this is what we are using. Border beam. https://magicui.design/docs/components/border-beam

Alright

**Comment by Simplypheyie96** [comment]
- Let's revisit the implementation of the circles. Currently, the size makes them overlap significantly, giving them a cluttered look. The previous implementation looked better than this. let's implement such that the circles won't overlap. I think that would be better if they are all orbiting. The bigger circle has covered up the dotted one too.
image

---

- this is what it looks like on mobile
image

---

- the gradient here is missing on the tablet view and on mobile view too.
image
image

---

- this section on the tablet view is not properly aligned. It's meant to be left-aligned. right now, it is left-aligned and centered at the same time which is giving it a weird look

image


Every other thing is spot on now.
@Damola18

**Review by Simplypheyie96 — APPROVED** [review]

**Comment by Tolu-Mals** [comment]
Screenshot 2024-07-02 at 10 26 35 AM

These circles should be transparent. I should be able to see the dashed lines through it.

Look at the design

Screenshot 2024-07-02 at 10 28 41 AM

Also the IQ logo should be bigger. And then the white outline circle and rounded rectangle should be smaller.

**Comment by Damola18** [comment]
> Screenshot 2024-07-02 at 10 26 35 AM
> 
> These circles should be transparent. I should be able to see the dashed lines through it.
> 
> Look at the design
> 
> Screenshot 2024-07-02 at 10 28 41 AM
> 
> Also the IQ logo should be bigger. And then the white outline circle and rounded rectangle should be smaller.

The zIndex used underlays or overlays all the two dashed lines because I am mapping through all of them due to the structure of the Orbit Circle.

**Comment by Simplypheyie96** [comment]
> Screenshot 2024-07-02 at 10 26 35 AM
> 
> These circles should be transparent. I should be able to see the dashed lines through it.
> 
> Look at the design
> 
> Screenshot 2024-07-02 at 10 28 41 AM
> 
> Also the IQ logo should be bigger. And then the white outline circle and rounded rectangle should be smaller.

I think the current state is acceptable. The assets are challenging to consolidate because they were separated by masking them out from the SVG. I had to mask them out.

Regarding the logo, we've reviewed it, and increasing its size further would make it too large on mobile devices. Unless there's a way to enlarge the logo on desktop without affecting the mobile/tablet view (@Damola18), we should keep the current size.

Also, I'm uncertain if we can make the circulating orbit look exactly like the design. The design is an SVG and isn't animated, which complicates replicating its exact appearance.

**Comment by Tolu-Mals** [comment]
> > Screenshot 2024-07-02 at 10 26 35 AM
> > These circles should be transparent. I should be able to see the dashed lines through it.
> > Look at the design
> > Screenshot 2024-07-02 at 10 28 41 AM
> > Also the IQ logo should be bigger. And then the white outline circle and rounded rectangle should be smaller.
> 
> I think the current state is acceptable. The assets are challenging to consolidate because they were separated by masking them out from the SVG. I had to mask them out.
> 
> Regarding the logo, we've reviewed it, and increasing its size further would make it too large on mobile devices. Unless there's a way to enlarge the logo on desktop without affecting the mobile/tablet view (@Damola18), we should keep the current size.
> 
> Also, I'm uncertain if we can make the circulating orbit look exactly like the design. The design is an SVG and isn't animated, which complicates replicating its exact appearance.

he can make the image responsive, it'll be larger on desktop, smaller on mobile.

File: public/locales/ko/about.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
   "AboutSEOTitle": "우리에 대해",
   "AboutSEODescription": "IQ 토큰을 통해 세계의 지식을 온체인으로 가져오는 것을 임무로 하는 세계 최대의 암호화 지식 기반입니다.",
+  "aboutHeroTitle": "IQ.Wiki 소개",
   "aboutHeroHeading": "블록체인 기술과 인공지능을 활용하여 암호화폐 지식을 확장합니다.",
   "aboutHeroPhrase": "IQ.wiki는 세계 최대 블록체인 및 암호화폐 백과사전으로, 암호화폐 공간의 모든 주제와 지식 분야에 대한 수천 개의 위키를 보유하고 있습니다.",
   "aboutSignUpBttn": "가입하기",
   "aboutgoTo": "아이큐 사이트로 이동",
+  "aboutFeatTitle": "",
```
Comment (Tolu-Mals) [inline public/locales/ko/about.json:9]: I think you forgot to add this translation

File: public/locales/ko/about.json [hunk 2]
```diff
@@ -16,29 +18,37 @@
...
   "latestFromBlogHeading": "블로그에서 최신 소식",
+  "historyTitle": "历史",
   "historyHeading": "우리의 역사에 대한 간략한 설명",
   "historyBrief": "세계 최대의 블록체인 백과사전은 단 한 순간에 탄생한 것이 아닙니다. IQ.wiki는 2014년에 \"모든 것의 백과사전\"으로 시작된 Everipedia로 출발하여 2018년에 IQ 토큰을 론칭한 후 세계 최초의 블록체인 백과사전이 되었습니다. 2022년에는 Everipedia가 Polygon 블록체인으로 확장되면서 IQ.wiki로 리브랜딩되었습니다.",
   "OurMission1": "저희는 현재 인터넷에서 블록체인 기술과 암호화폐에 관련된 모든 것에 대한 객관적이고 고품질의 지식의 주요 출처입니다. 우리의 비전은 전 세계가 블록체인을 더 쉽게 찾고 접근 가능하게 만들어 사용자들에게 신뢰할 수 있는 정보를 제공하는 것입니다.",
   "OurMission2": "투명성을 위해, IQ.wiki에서의 모든 편집은 폴리곤 블록체인에 기록되며 트랜잭션 해시는 온체인에 저장됩니다. 각 위키의 일부로 업로드된 데이터와 이미지는 IPFS 네트워크에 저장되며 해시는 사용자들이 검증할 수 있도록 각 페이지에 포함됩니다.",
   "OurMission3": "IQ.wiki는 또한 인공지능을 우리의 미션에 통합하여 전 세계에 블록체인 기술에 대한 교육을 제공하고 있습니다. 우리는 지식이 공유되기 쉽도록 콘텐츠를 확장하는 데 AI를 사용합니다. IQ.wiki는 위키 기사 요약을 포함한 작업에 OpenAI의 GPT-4를 통합합니다.",
   "OurMission4": "공간에 새로 오신 분이든, 전문 코더이든, 디파이 디젠이든, ETH 막시이든, 아니면 NFT 플리퍼이든, IQ.wiki는 암호화폐 공간에 관한 지식을 배우고 공유하는 곳입니다.",
-  "aboutNavinDescription": "나빈은 19세의 나이로 IQ.wiki(이전의 에버리피디아)에 창립 팀 멤버로 처음 합류하여, 커뮤니티 관리, 마케팅, 언론 및 국제 비즈니스 분야에서 여러 해 동안 경험을 쌓았습니다. 나빈은 아이비 비즈니스 스쿨에서 HBA 학위를 받았습니다.",
+  "aboutNavinDescription": "",
```
Comment (Tolu-Mals) [inline public/locales/ko/about.json:30]: The translation  here is empty as well

File: public/locales/zh/about.json [hunk 3]
```diff
@@ -1,42 +1,50 @@
 {
   "AboutSEOTitle": "关于",
   "AboutSEODescription": "世界上最大的加密知识库，其使命是通过IQ代币将世界的知识上链。",
+  "aboutHeroTitle": "关于 IQ.Wiki",
   "aboutHeroHeading": "我们使用区块链技术和人工智能扩展加密知识。",
   "aboutHeroPhrase": "IQ.wiki是世界上最大的区块链和加密货币百科全书，包含所有主题和加密空间知识分支的数千个wiki。",
   "aboutSignUpBttn": "注册",
   "aboutgoTo": "前往IQ网站",
   "aboutFeatHeading": "我们的使命是教育世界关于区块链技术",
   "aboutFeatContent": "我们是今天互联网上关于区块链技术和加密货币相关一切事物的首要知识来源。我们的愿景是使区块链在全世界更容易被发现和访问，并赋予用户他们可以信赖的信息。",
```
Comment (Tolu-Mals) [inline public/locales/zh/about.json:10]: `aboutFeatTitle` for chinese is missing here.

**Comment by Damola18** [comment]
> > > Screenshot 2024-07-02 at 10 26 35 AM
> > > These circles should be transparent. I should be able to see the dashed lines through it.
> > > Look at the design
> > > Screenshot 2024-07-02 at 10 28 41 AM
> > > Also the IQ logo should be bigger. And then the white outline circle and rounded rectangle should be smaller.
> > 
> > 
> > The zIndex used underlays or overlays all the two dashed lines because I am mapping through all of them due to the structure of the Orbit Circle.
> 
> I don't understand, but it's fine, if it's not possible

I meant the zIndex to control the position of the icon and the dashed line only goes any of the two ways

File: src/components/About/AboutFeatures.tsx [hunk 4]
```diff
@@ -15,56 +17,94 @@ const AboutFeatures = () => {
...
-        <AboutFeaturesCard
-          title={`${t('abtFeatThreeHeading')}`}
-          content={`${t('abtFeatThreeContent')}`}
-          icon={MdOutlinePeopleAlt}
-        />
-      </SimpleGrid>
-    </VStack>
+        <Flex
+          direction={{ base: 'column', md: 'row' }}
+          className="relative"
```
Comment (Tolu-Mals) [inline src/components/About/AboutFeatures.tsx:29]: I don't think we need this

File: src/components/About/AboutFeaturesCard.tsx [hunk 5]
```diff
@@ -14,19 +15,31 @@ const AboutFeaturesCard = ({
...
+    }
   >
-    <Icon as={icon} mb={4} w="40px" h="40px" />
-    <Heading fontWeight={700} fontSize={{ base: '18px', lg: '20px' }} mb={4}>
-      {title}
-    </Heading>
-    <Text fontWeight={500} fontSize={{ base: '14px', md: '16px', lg: '18px' }}>
-      {content}
-    </Text>
+    <MagicCard className="flex w-full cursor-pointer flex-col text-left overflow-hidden bg-white rounded-xl border border-gray200 dark:border-gray700">
```
Comment (Tolu-Mals) [inline src/components/About/AboutFeaturesCard.tsx:22]:
```suggestion
    <MagicCard className="flex w-full cursor-pointer flex-col text-left overflow-hidden bg-white rounded-xl border border-gray-200 dark:border-gray-700">
```

Did you mean to write this?

File: yarn.lock
Comment (Tolu-Mals) [inline yarn.lock:1]: we can add `yarn.lock` to `.gitignore`

**Comment by Damola18** [comment]
@Tolu-Mals @Simplypheyie96  made the changes as reviewed

**Comment by Damola18** [comment]
> you need to run this `git rm --cached yarn.lock` to stop `yarn.lock` from being tracked too. See if you can push after that.

Yeah, i just did

**Comment by Tolu-Mals** [comment]
please fix the build error


> > you need to run this `git rm --cached yarn.lock` to stop `yarn.lock` from being tracked too. See if you can push after that.
> 
> Yeah, i just did

please fix the build error

**Comment by Damola18** [comment]
> you need to run this `git rm --cached yarn.lock` to stop `yarn.lock` from being tracked too. See if you can push after that.

Checking vercel logs to find out what the build error is

**Review by Tolu-Mals — APPROVED** [review]

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**